christfellowshipchurch/web-app-v2
# Sisterhood Update

### About
This PR implements the Sisterhood Season Page when a Season is clicked on the `/so-good-sisterhood` page.

### Test Instructions
1. Run the API locally in the `rock-tags` branch.
2. Make sure your web-app is connected to the local API in the `.env` file.
3. Run the web-app.
4. Go to `/so-good-sisterhood`. The Podcast Seasons section is the only thing using Tags, and is being done in the services repo, not web-app-v2.
5. Click on a season, the new page should show up and clicking on individual episodes should redirect to the correct episode page.

### Screenshots

<img width="1439" alt="Screenshot 2023-09-18 at 10 48 09 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/72806945-1be7-44e1-ac1b-3e63a65829ad">
<img width="1440" alt="Screenshot 2023-09-18 at 10 53 50 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/70230d2c-0202-490f-90e9-61e2def09e7d">
<img width="1440" alt="Screenshot 2023-09-18 at 10 54 52 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/c77e39ce-0658-4d0d-9b4c-bcf208b61665">


### Closes Tickets

[CFDP-2730](https://christfellowshipchurch.atlassian.net/browse/CFDP-2730)

[CFDP-2730]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2730?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
